src/components/Main/AgentInput/index.js

```diff
@@ -18,17 +20,75 @@ class AgentInput extends Component {
       : this.setState({ relWords: value })
   }
 
-  handleSubmit = (e) => {
+  handleSendEvent(e) {
     e.preventDefault();
+      this.setState({
+      currentChatMessage: ''
+    });
+  }
+  renderChatLog() {
+    return this.state.chatLogs.map((chat) => {
```

Comment:
rendering this chatlog must be what you were mentioning earlier on how the previous hints would be like a constant chatbox

Reply:
Yep! We can definitely rename some things since its only sort of a chatbox